src/dotnet-publish-iis/Program.cs

```diff
@@ -15,46 +12,27 @@ public static int Main(string[] args)
         {
             var app = new CommandLineApplication
             {
-                // TODO: This needs to be updated once we know how this is going to be called
-                Name = "dotnet(????) publish-iis",
+                Name = "dotnet publish-iis",
                 FullName = "Asp.Net IIS Publisher",
                 Description = "IIS Publisher for the Asp.Net web applications",
             };
             app.HelpOption("-h|--help");
 
-            var publishFolderOption = app.Option("--publish-folder", "The path to the publish output folder", CommandOptionType.SingleValue);
-            var appNameOption = app.Option("--application-name", "The name of the application", CommandOptionType.SingleValue);
+            var publishFolderOption = app.Option("--publish-folder|-p", "The path to the publish output folder", CommandOptionType.SingleValue);
+            var webRootOption = app.Option("--webroot|-w", "The name of webroot folder", CommandOptionType.SingleValue);
```

Comment:
@Tratcher : how about: 

The name of webroot folder. Overwrites webroot from hosting.json file or the default "wwwroot"

?